tests/integration/routes/account/delete-account-test.js

```diff
@@ -99,7 +99,15 @@ getServer(function (error, server) {
   })
 
   test('DELETE /session/account?include=profile', {todo: true}, function (t) {
-    t.end()
+    var options = _.defaultsDeep({
+      url: '/session/account?include=profile'
+    }, routeOptions)
+
+    server.inject(options, function (response) {
+      t.is(response.statusCode, 400, 'returns 400 status')
+      t.deepEqual(response.result.errors[0].detail, 'Allowed value for ?include is \'profile\'', 'returns error message')
```

Comment:
it should not return an error. Instead it should return the body of the deleted account.